scripts/delayed.js

```diff
@@ -1,26 +1,73 @@
 /* eslint-disable import/no-cycle */
-import { getConfigValue } from './configs.js';
+import { FetchGraphQL } from '@dropins/tools/fetch-graphql.js';
+import { getHeaders } from './configs.js';
 import { getUserTokenCookie } from './initializers/index.js';
 import { getConsent } from './scripts.js';
+import { commerceEndpointWithQueryParams } from './commerce.js';
 
 async function initAnalytics() {
+  const gql = new FetchGraphQL(
+    await commerceEndpointWithQueryParams(),
+    await getHeaders('cs'),
+  );
+
+  // fetch data from storeconfig
+  const { data } = await gql.fetchGraphQl(`
+    query DSContextQuery {  
+      ds: dataServicesStorefrontInstanceContext {
```

Comment:
This fails against ACCS.

<img width="1341" alt="image" src="https://github.com/user-attachments/assets/7ad6add7-2f9a-4bda-80a8-708bf6ed0f93" />